easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.2.3.eb

```diff
@@ -0,0 +1,108 @@
+easyblock = 'Bundle'
+
+name = 'GC3Pie'
+version = '2.2.3'
+
+homepage = 'https://gc3pie.readthedocs.org'
+description = """GC3Pie is a Python package for running large job campaigns on diverse batch-oriented execution
+ environments."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ('python -c"import %(ext_name)s"', '')
+
+# allow use of system Python
```

Comment:
well, without this, the build won't work because `get_software_root('Python')` will return `None`